persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/StargateSystemKeyspace.java

```diff
@@ -196,4 +203,20 @@ public static void forceBlockingFlush(String cfname) {
           Keyspace.open(SYSTEM_KEYSPACE_NAME).getColumnFamilyStore(cfname).forceFlush());
     }
   }
+
+  public static Set<String> generateRandomTokens(InetAddress inetAddress, int numTokens) {
+    Random random = new Random(getSeed(inetAddress));
```

Comment:
if this is on the hot path maybe we should extract the `new Random()` to constant. 
Is it called by one thread? If not maybe `ThreadLocalRandom` would be a better choice

Reply:
It's very rarely on the hot path.